ui/app/app.js

```diff
@@ -52,6 +52,14 @@ export default class App extends Application {
         },
       },
     },
+    ldap: {
+      dependencies: {
+        services: ['router', 'store', 'secret-mount-path', 'flash-messages'],
```

Comment:
These services are commonly used within secrets engine workflows so I went ahead and added them now.